pkg/controllers/controllers.go

```diff
@@ -52,10 +53,11 @@ func NewControllers(
 
 	p := provisioning.NewProvisioner(kubeClient, kubernetesInterface.CoreV1(), recorder, cloudProvider, cluster)
 	evictionQueue := terminator.NewQueue(kubernetesInterface.CoreV1(), recorder)
+	disruptionQueue := orchestration.NewQueue(kubeClient, recorder, cluster, clock, p)
```

Comment:
Can we cut a followon OE task to model these queues as controllers? We can include this as part of the v1 OE work to standardize metrics and automate alarms.

Reply:
Good call. https://github.com/aws/karpenter-core/issues/777